src/funnel.hpp

```diff
@@ -143,6 +154,9 @@ class Funnel {
     /// Dump information from the Funnel as a dot-format Graphviz graph to the given stream.
     /// Illustrates stages and provenance.
     void to_dot(ostream& out);
+
+    /// Set an alignments annotations with the number of results at each stage
```

Comment:
This should probably explain what track_correctness means. It also might want to be renamed for this function, because it doesn't actually control the tracking of correctness here.